packages/service-library/src/servicelib/long_running_tasks/_task.py

```diff
@@ -316,8 +317,14 @@ async def remove_task(
         reraise_errors: bool = True,
     ) -> None:
         """cancels and removes task"""
+        logger.debug(
+            "Attempting to remove task with task_id=%s. Stack trace:\n%s",
+            task_id,
+            "".join(tb.format_stack()),
```

Comment:
is there not a argument called exc_info, just put ```exc_info=True``` that should do the trick

Reply:
agree with SAN